### PR TITLE
Cypress test. Add/delete labels and attributes.

### PR DESCRIPTION
**Pull request by dvkruchinin**
### Motivation and context
Add Cypress test. Add/delete labels and attributes.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: tests/cypress/integration/actions_tasks_objects/case_41_add_delete_label_attribute.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    const textDefaultValue = 'Some default value for type Text';
+
+    before(() => {
+        cy.visit('auth/login');
+        cy.login();
+    });
+
+    describe(`Testing "${labelName}"`, () => {
+        it('Go to create task page. Open label constructor.', () => {
+            cy.get('#cvat-create-task-button').click();
```
Comment (bsekachev) [inline tests/cypress/integration/actions_tasks_objects/case_41_add_delete_label_attribute.js:20]: Probably should be a part of ``before``

File: tests/cypress/integration/actions_tasks_objects/case_41_add_delete_label_attribute.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            cy.get('.cvat-constructor-viewer-new-item').click();
```
Comment (bsekachev) [inline tests/cypress/integration/actions_tasks_objects/case_41_add_delete_label_attribute.js:21]: Probably should be a part of next test

File: tests/cypress/integration/actions_tasks_objects/case_41_add_delete_label_attribute.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            cy.get('.cvat-new-attribute-button').click();
+            cy.get('.cvat-attribute-name-input').type(attrName);
+            cy.get('.cvat-attribute-type-input').click();
+            cy.get('.cvat-attribute-type-input-text').click();
+            cy.get('.cvat-attribute-values-input').type(textDefaultValue);
+            cy.contains('[type="submit"]', 'Done').click();
+            cy.get('.cvat-constructor-viewer-item').should('exist');
+        });
+
+        it('Start to edit the label. Attribute should be exist. Remove the atrribute. Press Done.', () => {
```
Comment (bsekachev) [inline tests/cypress/integration/actions_tasks_objects/case_41_add_delete_label_attribute.js:54]: be exist >> exist

File: tests/cypress/integration/actions_tasks_objects/case_41_add_delete_label_attribute.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+            cy.get('.cvat-attribute-inputs-wrapper')
+                .should('exist')
+                .within(() => {
+                    cy.get('.cvat-delete-attribute-button').click();
+                });
+            cy.get('.cvat-attribute-inputs-wrapper').should('not.exist');
+            cy.contains('[type="submit"]', 'Done').click();
+        });
+
+        it('Start to edit the created label. Attribute should not be exist. Press Cancel.', () => {
```
Comment (bsekachev) [inline tests/cypress/integration/actions_tasks_objects/case_41_add_delete_label_attribute.js:65]: be exist >> exist

File: tests/cypress/integration/actions_tasks_objects/case_41_add_delete_label_attribute.js [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+            cy.get('.cvat-constructor-viewer-item').find('[aria-label="edit"]').click();
+            cy.get('.cvat-attribute-inputs-wrapper').should('not.exist');
+            cy.contains('[type="button"]', 'Cancel').click();
```
Comment (bsekachev) [inline tests/cypress/integration/actions_tasks_objects/case_41_add_delete_label_attribute.js:68]: Let it be a part of the previous ``it``

**Review by bsekachev — CHANGES_REQUESTED** [review]

File: tests/cypress/integration/actions_tasks_objects/case_41_add_delete_label_attribute.js (hunk 1 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/case_41_add_delete_label_attribute.js]: Yes, it's more visual. Thanks.

File: tests/cypress/integration/actions_tasks_objects/case_41_add_delete_label_attribute.js (hunk 2 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/case_41_add_delete_label_attribute.js]: Moved it.

File: tests/cypress/integration/actions_tasks_objects/case_41_add_delete_label_attribute.js (hunk 3 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/case_41_add_delete_label_attribute.js]: Done.

File: tests/cypress/integration/actions_tasks_objects/case_41_add_delete_label_attribute.js (hunk 4 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/case_41_add_delete_label_attribute.js]: Done.

File: tests/cypress/integration/actions_tasks_objects/case_41_add_delete_label_attribute.js (hunk 5 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/case_41_add_delete_label_attribute.js:63]: Moved it.

**Review by bsekachev — APPROVED** [review]

**Pull request merged**